addons/l10n_nl/migrations/3.1/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,6 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.addons.account.models.chart_template import update_taxes_from_templates
+
+
+def migrate(cr, version):
+    update_taxes_from_templates(cr, 'l10n_nl.l10nnl_chart_template')
```

Comment:
I don't think you need this anymore in 16.2 ^^ The template update is now done with a button in the settings ; no local migration script needed.